api/app/Traits/Generator/GeneratesUserDoc.php

```diff
@@ -422,6 +427,33 @@ public function experience(Section $section, AwardExperience|CommunityExperience
                         $classification ? $classification->group.'-'.$classification->level : Lang::get('common.not_found', [], $this->lang),
                     );
                 }
+                $this->addLabelText($section, $this->localize('experiences.supervisory_position'), $this->yesOrNo($experience->supervisory_position));
+                if ($experience->supervisory_position === true) {
+                    $this->addLabelText($section, $this->localize('experiences.supervised_employees'), $this->yesOrNo($experience->supervised_employees));
+                    if ($experience->supervised_employees === true) {
+                        $this->addLabelText(
+                            $section,
+                            $this->localize('experiences.supervised_employees_number'),
+                            $experience->supervised_employees_number
+                        );
+                    }
+                    $this->addLabelText($section, $this->localize('experiences.budget_management'), $this->yesOrNo($experience->budget_management));
+
+                    if ($experience->budget_management === true) {
+                        $this->addLabelText(
+                            $section,
+                            $this->localize('experiences.annual_budget_allocation'),
+                            Number::format($experience->annual_budget_allocation, precision: 0, locale: App::getLocale()), // TODO: missing dollar sign
+                        );
+                    }
+                    $this->addLabelText($section, $this->localize('experiences.senior_management_status'), $experience->senior_management_status ? Lang::get('experiences.senior_management_true') : Lang::get('experiences.senior_management_false'));
+                    if ($experience->senior_management_status === true) {
+                        $this->addLabelText($section, $this->localize('experiences.c_suite_role_title'), $this->localizeEnum($experience->c_suite_role_title, CSuiteRoleTitle::class));
+                    }
+                    if ($experience->c_suite_role_title === CSuiteRoleTitle::OTHER->name) {
+                        $this->addLabelText($section, $this->localize('experiences.other_c_suite_role_title'), $this->localize('experiences.other_c_suite_role_title'));
+                    }
```

Comment:
This one is a bit broken at the moment, should just render the text field. No localization needed

![image](https://github.com/user-attachments/assets/c9def210-0bfc-4469-9c51-0493703bcc10)